libs/sdk-ui-filters/src/RankingFilter/MeasureDropdown/MeasureDropdownItem.tsx

```diff
@@ -11,6 +11,7 @@ interface IMeasureDropdownItemProps {
     onSelect: (ref: ObjRefInScope) => void;
```

Comment:
Please update the copyright of this file as well. As we are making some changes now, the copyright should change accordingly, shouldn't it?

`// (C) 2020 GoodData Corporation`

↓

`// (C) 2020-2022 GoodData Corporation`

Reply:
fixed